cmake/Toolchain-mingw32.cmake

```diff
@@ -4,7 +4,7 @@
 
 set(CMAKE_SYSTEM_NAME Windows)
 
-if(NOT COMPILER_PREFIX)
+if(COMPILER_PREFIX STREQUAL "")
```

Comment:
I think the intention here is to check whether the variable is defined or not.
Could you test something like this?
```
if(NOT DEFINED COMPILER_PREFIX)
```

Reply:
Hello! Thanks for reviewing. I tried this, but unfortunately it did not work. Only checking for non-empty string did.

Reply:
Ok thanks for testing and feedback. I will merge as is.